packages/angular_devkit/build_webpack/test/browser/replacements_spec_large.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { join, normalize, virtualFs } from '@angular-devkit/core';
+import { tap } from 'rxjs/operators';
+import { browserTargetSpec, host, runTargetSpec, workspaceRoot } from '../utils';
+
+
+describe('Browser Builder', () => {
+  const outputPath = normalize('dist');
+
+  beforeEach(done => host.initialize().subscribe(undefined, done.fail, done));
+  afterEach(done => host.restore().subscribe(undefined, done.fail, done));
+
+  it('allows file replacements', (done) => {
+    host.writeMultipleFiles({
+      'src/meaning-too.ts': 'export var meaning = 42;',
+      'src/meaning.ts': `export var meaning = 10;`,
+
+      'src/main.ts': `
+        import { meaning } from './meaning';
+
+        console.log(meaning);
+      `,
+    });
+
+    const overrides = {
+      fileReplacements: [
+        {
+          from: join(workspaceRoot, normalize('/src/meaning.ts')),
```

Comment:
This path should be relative to workspace? So just `normalize('/src/meaning.ts')`.

Reply:
Done